src/main/proto/jwt_service.proto

```
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package com.vinaacademy.grpc;
```

Buf (1.55.1)

Files with package "com.vinaacademy.grpc" must be within a directory "com/vinaacademy/grpc" relative to root but were in directory "src/main/proto".

(PACKAGE_DIRECTORY_MATCH)